dWorldServer/WorldServer.cpp

```diff
@@ -313,8 +313,9 @@ int main(int argc, char** argv) {
 		if (!Game::server->GetIsConnectedToMaster()) {
 			framesSinceMasterDisconnect++;
 
-			if (framesSinceMasterDisconnect >= 30) {
-				Game::logger->Log("WorldServer", "Game loop running but no connection to master for 30 frames, shutting down");
+			int framesToWaitForMaster = ready ? 10 : 200;
+			if (framesSinceMasterDisconnect >= framesToWaitForMaster && !worldShutdownSequenceStarted) {
+				Game::logger->Log("WorldServer", "Game loop running but no connection to master for %d frames, shutting down", framesToWaitForMaster);
```

Comment:
Should probably be a new line here

Reply:
In the log message or after the call to `Log`? I'm not sure why you would want a newline? Width of terminals? C++ formatter rules?

Reply:
I mean in the log - I noticed in a screenshot that the log just filled up the whole terminal window
![image](https://user-images.githubusercontent.com/39972741/152976184-316eb297-a667-49b3-8912-32c447430f7f.png)

Reply:
There should indeed be a "\n" at the end of the log format

Reply:
Ah, now I get it, thanks!